.circleci/config.yml

```diff
@@ -126,6 +151,8 @@ workflows:
       - check_mergable
       - test_71_mysql:
           <<: *requires
+      - test_72_drupal88:
+          <<: *requires
       - test_72_highest:
```

Comment:
We could rename this "drupal90" if desired.

Reply:
If it downloads Druapl 9, that rename seems wise.